tests/test_bot.py

```diff
@@ -365,12 +365,17 @@ async def test_get_me_and_properties_not_initialized(self, bot: Bot, attribute):
         finally:
             await bot.shutdown()
 
-    async def test_get_me_and_properties(self, bot: Bot):
-        get_me_bot = await bot.get_me()
+    async def test_get_me_and_properties(self, bot):
+        mocked = "mocked"
+        with bot._bot_user._unfrozen():
+            bot._bot_user.username = mocked  # set the attribute to a mocked value
+        assert bot.username == mocked  # To test we are using the mocked bot
+        # call the parent class get_me method instead of the overriden get_me method
+        get_me_bot = await super(bot.__class__, bot).get_me()
```

Comment:
see above. that way we also don't have a need for using this mocking setup, which tbh seems rather convoluted to me …